packages/cli/src/lib/snapshot/reportPreviewer/reportPreviewerDataModels.ts

```diff
@@ -7,3 +7,9 @@ export interface ReportViewerResourceReportModel {
   name: string;
   operations: ResourceSnapshotsReportOperationModel;
 }
+
+export enum SnapshotReportStatus {
```

Comment:
Could we have that in the Platform-client?

Reply:
Currently, it would not make sense because the REST API does not return anything that relates to that. When the contract for the new reports will be defined, yes prolly.
Although, we would certainly have some kind of extension (I'm adding a new ReportStatus for the "no changes")